src/main/java/com/foo/gosucatcher/domain/estimate/domain/MemberRequestEstimate.java

```diff
@@ -53,4 +60,10 @@ public MemberRequestEstimate(Member member, SubItem subItem, String location, Lo
 		this.startDate = startDate;
 		this.detailedDescription = detailedDescription;
 	}
+
+	public void update(MemberRequestEstimate memberRequestEstimate) {
+		this.location = memberRequestEstimate.getLocation();
+		this.startDate = memberRequestEstimate.getStartDate();
```

Comment:
```suggestion
		this.startDate = validateStartDate(memberRequestEstimate.getStartDate());
```
```
//엔티티로 이동
private void validateStartDate(LocalDateTime startDate) {
		if (LocalDateTime.now().isAfter(startDate)) {
			throw new BusinessException(ErrorCode.INVALID_START_DATE);
		}
	}
```
이렇게 하면 엔티티 안으로 캡슐화 할 수 있을 거 같은 느낌적인 느낌..!?

Reply:
수정했습니다!